h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -997,6 +1004,15 @@ else if (s.equals("-proxy")) {
     return otherArgs;
   }
 
+  /**
+   *
+   * @return True if current Java version uses unified logging (JEP 158), otherwise false.
+   */
+  boolean useUnifiedLogging(){
```

Comment:
Maybe we could move this function to JavaVersionUtils? IMHO it would be good to keep this all in one place.